src/scss/components/carousel.scss

```diff
@@ -179,10 +179,10 @@
   padding-right: 6px;
 
   .btn-link {
-    color: $carousel-text-color;
+    color: $carousel-caption-color;
 
     &:hover {
-      color: rgba($carousel-text-color, .8);
+      color: rgba($carousel-caption-color, .8);
     }
   }
 }
```

Comment:
For the carousel-mask below the mask colour is not coming out the same as before - is that intended?
It is down to the '$carousel-mask-bg-color' variable being different based on black rather than the theme colour.